cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -284,6 +327,28 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
     }
 
+    /**
+     * Retrieve the license categories for the [package][pkg] based on its
+     * [effective license][ResolvedLicenseInfo.effectiveLicense].
```

Comment:
Nit: You can have a line break between `[effective license]` and `[ResolvedLicenseInfo.effectiveLicense]` without breaking the link.